src/cloud/docker/docker-manage-cron-jobs.md

```diff
@@ -19,6 +19,14 @@ The Magento cron implementation has the following limitations:
 -  The `setup:cron:run` and `cron:update` commands are not available on Cloud and Docker for Cloud environments
 -  In the Docker environment, cron works only with the CLI container to run the `./bin/magento cron:run` command
 
+By default, cron container is not present in your Docker development environment to improve overall performance.
```

Comment:
```suggestion
To improve the overall performance in the Docker development environment, the cron container is not present by default. You can add the cron container as needed using the `--with-cron` option.
```

Reply:
We have an overlap of suggestions here.  My suggested edit is:

By default, the Docker build configuration does not include the Cron container. This configuration improves overall performance in the Docker production and development environments. You have the option to add the Cron container when you build the Docker environment.

It doesn't matter which we use, but I think it's important to qualify that the Cron container is not included in either the Docker production or developer  environments.